src/SectionMeetings.tsx

```diff
@@ -0,0 +1,163 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import * as Moment from 'moment';
+import { IMeeting } from './types';
+
+interface Props {
+  meetings: Array<IMeeting>;
+}
+
+const styles = {
+  container: {
+    flex: 1,
+    wordWarp: 'normal',
+    justifyContent: 'flex-start',
+    div: {
+      display: 'inline-block',
+    },
+  },
+  boldedText: {
+    fontWeight: 700,
+  },
+};
+
+const SectionMeetings: React.FC<Props> = ({ meetings }) => {
+  // Correct for null meeting arrays
+  let correctedMeetings = meetings ?? [];
+  let sortedSectionMeetings = correctedMeetings.sort((a, b) =>
+    Moment.default(a.beginDate).isBefore(Moment.default(b.beginDate)) ? 1 : -1
```

Comment:
Any reason we're doing all of this instead of `a.beginDate.valueOf() - b.beginDate.valueOf()`? I'm not super familiar with moment

Reply:
For some reason I couldn't get that method to work originally, probably something I was doing wrong